api/app/src/mappings/fitbit/body.ts

```diff
@@ -56,8 +56,10 @@ export const mapToBody = (
   return body;
 };
 
-function checkSource(weight: string) {
-  return weight === "API" ? SourceType.manual : SourceType.device;
+function checkSource(weight_src: string) {
+  return weight_src.toLowerCase() === "api" || weight_src.toLowerCase() === "web"
+    ? SourceType.manual
+    : SourceType.device;
```

Comment:
so you don't have to do the conversion twice - this would be better:
```
const weightSrcLower = weight_src.toLowerCase();
return weightSrcLower === "api" || weightSrcLower === "web"
    ? SourceType.manual
    : SourceType.device;
```

Reply:
Fixed